### PR TITLE
Update zoning.md

### PR DESCRIPTION
**Pull request by aaron-s-weber**
Jeff - could you review these proposed changes/updates to the About Zoning page?

### CODE REVIEW

File: resources/zoning.md [hunk 1]
```diff
@@ -38,36 +38,38 @@ In Somerville and many other places in the US, zoning got a lot stricter over th
...
+The most common districts in the city are:
 
-* Neighborhood Residence: Mostly 1–2 family residential buildings.
-* Urban Residence: Mostly multifamily residential buildings, with some corner stores and small retail.
-* Midrise 3, 4, and 5: 3-5 story buildings, mostly commercial. Large apartment buildings allowed by special permit.
-* Commercial Core 3, 4, and 5: 3-5 story commercial buildings.
-* High Rise: Tall buildings.
+* Neighborhood Residence (NR): One and two-family residential buildings, with the occasional triple-decker.
+* Urban Residence (UR): Multifamily residential buildings such as triple-deckers, row houses, and small-to-medium apartment buildings.
+* Midrise 3, 4, 5 and 6 (MR3, MR4, MR5, MR6): 3-6 story buildings, such as large apartment buildings with retail on the ground floor.
```
Comment (jeffbyrnes) [inline resources/zoning.md:45]: I think we have MR7 too?

File: resources/zoning.md [hunk 2]
```diff
@@ -38,36 +38,38 @@ In Somerville and many other places in the US, zoning got a lot stricter over th
...
+* High Rise (HR): Tall buildings (7 or more stories).
 
-We also have overlay districts, which modify the base rules for specific reasons, ranging from flood risks to special development areas like Assembly Square. One of our favorites is the Affordable Housing Overlay (AHO), which allows more height and faster permitting for projects where every home is affordable. For more information about the exact definition of affordability and how we achieve it in Somerville, see [“What exactly do we mean by Affordable Housing?”](/resources/affordable-housing/)
+We also have overlay districts, which modify the base rules for specific reasons. For example, the Flood Zone overlay requires additional flood control measures, and the Small Business Overlay allows corner stores and other light commercial activities in parts of NR and UR. One of our favorites is the Affordable Housing Overlay (AHO), which allows more height and faster permitting for projects where every home is affordable. For more information about the exact definition of affordability and how we achieve it in Somerville, see [“What exactly do we mean by Affordable Housing?”](/resources/affordable-housing/).
```
Comment (jeffbyrnes) [inline resources/zoning.md]: Maybe better to say

> “…where every home is subsidized affordable…”

b/c these are specifically for subsidized projects.

File: resources/zoning.md [hunk 3]
```diff
@@ -38,36 +38,38 @@ In Somerville and many other places in the US, zoning got a lot stricter over th
...
+Our least-dense district is far better than the “single-family only” rules seen in other cities, but it could be better. For example, it should allow [accessory dwelling units](https://www.mass.gov/service-details/smart-growth-smart-energy-toolkit-modules-accessory-dwelling-units-adu) (“in-law apartments”) and three-family buildings like our iconic triple-deckers with far fewer restrictions.
 
 ### Midrise Districts
 
-We currently require special permits to build apartments in midrise districts, but should allow that use by right. These permits have generally been granted, but there’s not a really good reason to require them at all.
+We currently require special permits to build apartments in midrise districts, but should allow that use by right. These city usually grants these permits when requested, but there’s not a really good reason to require them at all.
```
Comment (jeffbyrnes) [inline resources/zoning.md]: Grammar fix:

> “These city usually grants these permits when requested…”

should be

> “The city usually grants these permits when requested…”

File: resources/zoning.md [hunk 4]
```diff
@@ -38,36 +38,38 @@ In Somerville and many other places in the US, zoning got a lot stricter over th
...
 * **Davis Square**: Most of the center of Davis Square is zoned for Midrise 3. Something so close to the Red Line deserves Commercial Core or Midrise 6, with surrounding residential neighborhoods zoned for Urban Residence. This is especially important as an equity issue, because the eastern parts of the city have seen far more upzoning, and Davis Square shouldn’t be exempt from development.
-* **Areas surrounding the new Green Line Extension stops**: Many areas near new Green Line stops are currently zoned for Neighborhood Residence and Urban Residence, but really ought to be allowed to grow to 4 to 6 stories high.
+* **Areas surrounding Green Line Extension stops**: Many areas near new Green Line stops are currently zoned for Neighborhood Residence or Urban Residence. In the spirit of the [MBTA Communities Law]([url](https://www.mass.gov/info-details/multi-family-zoning-requirement-for-mbta-communities)), we should adjust the map to increase allowable density near transit.
```
Comment (jeffbyrnes) [inline resources/zoning.md]: Markdown formatting fix, your hyperlink should be like so:

```markdown
spirit of the [MBTA Communities Law](https://www.mass.gov/info-details/multi-family-zoning-requirement-for-mbta-communities)
```

**Review by jeffbyrnes — CHANGES_REQUESTED** [review]
Looks pretty good! Some suggestions to tighten it up & fix a few things.

File: resources/zoning.md (hunk 1 above)
Comment (aaron-s-weber) [reply resources/zoning.md:45]: No, just up to 6. At 7 it counts as HR.

**Review by aaron-s-weber — COMMENTED** [review]
OK, changes made! Can you confirm and approve?

**Review by jeffbyrnes — APPROVED** [review]

**Pull request merged**